### ISSUE TITLE
Support for 5-character ISO codes

### ISSUE THREAD

**Issue by hbjydev** [ultrabug/mkdocs-static-i18n#37]
Currently, languages like traditional Chinese (`zh`) are working fine.
But other languages, like simplified Chinese (`zh_CN`) and simplified Chinese for Taiwan (`zh_TW`), etc. just break the system.

```
➜  rldocs git:(main) ✗ python3 -m mkdocs serve
INFO     -  Building documentation...
ERROR    -  Config value: 'plugins'. Error: Plugin 'i18n' value: 'languages'. Error: Language values must respect the ISO-639-1 (2-letter)
            format, received 'zh-cn' of length '5'.
Aborted with 1 Configuration Errors!
```

**Comment by ultrabug** [comment]
@hbjydev hello

Indeed, I applied this rule when I was contributing localization support to mkdocs themes and we decided to support longer country codes in the end but I did not apply it here.

My only concern is that this plugin automatically configures the search plugin and mkdocs-material languages but they do not support all the lang combinations... For instance, mkdocs-material only supports ISO-639-1 country codes.

This plugin already tries to be smart about this and issues warning logs when the underlying project or plugin do not support the given language, I guess you'll be fine with this behavior?

**Comment by hbjydev** [comment]
Yeah, the languages in question don't have too much content for now, and if I need to look into contributing to the upstream projects to fix that, that's completely fine by me :)

**Comment by ultrabug** [comment]
Ok I will make that happen then mate, stay tuned

**Comment by hbjydev** [comment]
Awesome, thanks! :)

**Comment by ultrabug** [comment]
```
WARNING  -  Language 'zh_TW' is not supported by lunr.js, not setting it in the 'plugins.search.lang' option
INFO     -  Cleaning site directory
INFO     -  Building documentation to directory: /home/alexys/github/mkdocs-static-i18n/site
INFO     -  Building en documentation
INFO     -  Building zh_TW documentation
WARNING  -  Language zh_TW is not supported by mkdocs-material==7.1.0, not setting the 'theme.language' option
```

Okay, you would get that kind of output, I'm almost done

**Comment by ultrabug** [comment]
available in version 0.16 mate, enjoy :star:
